tests/main/prepare-image-grub-core18/task.yaml

```diff
@@ -5,12 +5,9 @@ backends: [-autopkgtest]
 systems: [-ubuntu-core-16-*, -fedora-*, -opensuse-*, -arch-*]
 
 environment:
-    ROOT: /tmp/root
-    IMAGE: /tmp/root/image
-    GADGET: /tmp/root/gadget
-
-restore: |
-    rm -rf "$ROOT"
```

Comment:
this seems to be fine, since the global per-test restore for the main suite will delete everything in `$PWD` right ?